libs/vertexai/langchain_google_vertexai/chat_models.py

```diff
@@ -709,11 +709,14 @@ def _prepare_request_gemini(
         functions: Optional[List[_FunctionDeclarationLike]] = None,
         tool_config: Optional[Union[_ToolConfigDict, ToolConfig]] = None,
         safety_settings: Optional[SafetySettingsType] = None,
+        response_mime_type: Optional[str] = None,
```

Comment:
I don't think we need this line anymore, do we? it will be passed as a part of `kwargs`.

Reply:
removed